spec/unit/block_score/actions/create_spec.rb

```diff
@@ -9,4 +9,18 @@
     it { is_expected.to be_persisted }
     it { is_expected.to be_an_instance_of BlockScore::Candidate }
   end
+
+  describe '.included' do
```

Comment:
You could probably move this into a shared example and reuse it in the affected specs.